### ISSUE TITLE
[Feature] Apply filter in products

### ISSUE THREAD

**Issue by dhruv8433** [pooranjoyb/popShop#246]
### Description

@pooranjoyb 

we can apply filter and based on that the products will be display

### Screenshots

![image](https://github.com/pooranjoyb/popShop/assets/114583978/e2071196-a3f7-4064-a952-7d8aa007fa25)

### Checklist

- [X] I have checked the existing [issues](https://github.com/pooranjoyb/popShop/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/pooranjoyb/popShop/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

**Comment by pooranjoyb** [comment]
Congratulations, @dhruv8433! 🎉 Thank you for creating your issue. Your contribution is greatly appreciated and we look forward to working with you to resolve the issue. Keep up the great work!<p>We will promptly review your changes and offer feedback. Keep up the excellent work! Kindly remember to check our [contributing guidelines](https://github.com/pooranjoyb/popShop/blob/dev/CONTRIBUTING.md)</p>

**Comment by pooranjoyb** [comment]
Yes go ahead with this. Also update the DB with amount to be in INR @dhruv8433

**Comment by dhruv8433** [comment]
Ok @pooranjoyb
